### PR TITLE
fixes jquery conflict with other version

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- fixes jquery conflict if other jquery version is loaded for example with another theme.
- wrapped tree_component into function to assign $ to CMS.$ and loads tree_component function globally.
- Because there were jQuery and $ mixed in tree_component I replaced them with $ to make it consistent

### CODE REVIEW

File: cms/templates/admin/cms/page/change_form.html [hunk 1]
```diff
@@ -12,9 +12,13 @@
 
 {% block extrahead %}
 {{ block.super }}
+<!-- load nonconflict pre and post files on line 17 and 21 to fix conflict if there is another jQuery loaded -->
```
Comment (vxsx) [inline cms/templates/admin/cms/page/change_form.html]: don't use html comments, use django comments instead.
also, line numbers tend to change so no need to specify them

**Comment by vxsx** [comment]
shame that github is not showing the changes in tree_component

**Comment by FinalAngel** [comment]
We would have to test this extensively but makes sense to implement :+1:

**Comment by vxsx** [comment]
:+1:

**Pull request merged**